python/src/viewer.py

```diff
@@ -576,32 +632,70 @@ def save(self, fname, **kwargs):
             A string containing a path to a filename from which file format is deduced.
 
         kwargs:
-            Refer to matplotlib.figure.Figure.savefig documentation for valid keyword arguments.
+            See https://matplotlib.org/api/_as_gen/matplotlib.figure.Figure.html#matplotlib.figure.Figure.savefig documentation for valid keyword arguments.
+        
+        Examples
+        --------
+        >>> import openturns as ot
+        >>> from openturns.viewer import View
+        >>> graph = ot.Normal().drawPDF()
+        >>> view = View(graph)
+        >>> view.save('graph.png', dpi=100)
         """
         self._fig.savefig(fname, **kwargs)
 
     def getFigure(self):
         """
         Accessor to the underlying figure object.
 
-        Refer to matplotlib.figure.Figure for further information.
+        See https://matplotlib.org/api/_as_gen/matplotlib.figure.Figure.html#matplotlib.figure.Figure for further information.
```

Comment:
Same remark again

Reply:
Ok. I remove all links to the Matplotlib website.